packages/models-library/src/models_library/services.py

```diff
@@ -482,6 +483,11 @@ class ServiceDockerData(ServiceKeyVersion, _BaseServiceCommonDataModel):
         alias="boot-options",
         description="Service defined boot options. These get injected in the service as env variables.",
     )
+    min_visible_inputs: NonNegativeInt | None = Field(
+        None,
+        alias="min-visible-inputs",
+        description="Service defined boot options. These get injected in the service as env variables.",
```

Comment:
This description needs update (I guess was just copied-and-pasted from above).

Reply:
It should be better now

Reply:
Yes, apart from a typo. 😉